compiler/optimizations/remoteValueForwarding.cpp

```diff
@@ -84,6 +88,83 @@ void remoteValueForwarding() {
   }
 }
 
+static void fixLHSArray(CallExpr* move, std::vector<Symbol*>& todo) {
+  Symbol* LHS = toSymExpr(move->get(1))->symbol();
+  if (LHS->isRef()) {
+    LHS->type = LHS->getValType();
+    LHS->qual = QUAL_VAL;
+    todo.push_back(LHS);
+  }
+}
+
+static void replaceRecordWrappedRefs() {
+  std::vector<Symbol*> todo;
+  forv_Vec(AggregateType, aggType, gAggregateTypes) {
```

Comment:
Let's put a comment above this for loop that says something like

      // Change fields storing references to record wrapped types into values
      // In particular note that this will modify argument bundle classes